### PR TITLE
KAA-1611: [Reviewed] Using C++ SDK, Linux. [master]

### PR DESCRIPTION
**Pull request by hunch7**

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md [hunk 1]
```diff
@@ -172,53 +187,52 @@ src/
...
+   ```
 
 ### Code
 
-The code for this demo is simple and straightforward. It just initializes and starts Kaa client,
-which involves connecting to Kaa server. After the `kaaClient->start();` line, Kaa client is up and running in a dedicated thread.
-Finally, the Endpoint access token is printed to `stdout` and Kaa client stops its execution.
+Below is a simple and straightforward example of the application code.
+
+The application will just initializes and starts Kaa client, which involves connecting to [Kaa server]({{root_url}}Glossary/#kaa-server).
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md]: will just initialize~~s~~

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md [hunk 2]
```diff
@@ -172,53 +187,52 @@ src/
 kaa/
 ```
 
-* `CMakeLists.txt` describes your application to CMake build system (see below).
-* `src/KaaDemo.cpp` is the actual application source code.
-* `kaa/` is a directory where the Kaa SDK should be placed.
-<!-- TODO: KAA-700 -->
-Download generated SDK archive from [Administration UI]({{root_url}}Glossary/#administration-ui) and unpack it to `kaa/` directory.
+* `CMakeLists.txt` -- file describing your application for the CMake [build system](#build-system-overview).
```
Comment (Polinaa) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md]: bad link

**Review by Polinaa — APPROVED** [review]

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md (hunk 1 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md]: Fixed.

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md (hunk 2 above)
Comment (hunch7) [reply doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/SDK-Linux/index.md]: Fixed.

**Review by vadimol — APPROVED** [review]

**Pull request merged**